.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+jobs:
+  build:
+    parallelism: 1
+    working_directory: ~/ezcater
+    docker:
+      - image: circleci/ruby:2.4
+    steps:
+      - checkout
+
+      # Restore bundle cache
+      - restore_cache:
+          keys:
+            - ruby-cache-{{ .Branch }}-{{ checksum "Gemfile" }}
```

Comment:
For a gem you also want to include `checksum X.gemspec` here. Probably not necessary to add additional fallbacks based on that.

Reply:
I'm just trying to be consistent with the fallbacks we use in other configurations... didn't see a reason to stray from the template we already kind of have in place. That said, will replace the Gemfile check with the gemspec check.